app/build.gradle

```diff
@@ -22,23 +23,23 @@ android {
     }
 
     buildTypes {
-        debug {
-            applicationIdSuffix = ".debug"
-        }
-
         release {
-            zipAlignEnabled true
+            minifyEnabled true
```

Comment:
What does this do? This won't enable proguard, will it?

Reply:
minifyEnabled will, but zipAlignEnabled isn't needed because it's already set to true for release builds by default.

Reply:
The thing is, I never tested this plugin with proguard, can you add `useProguard false` then?

Reply:
Uhh sure, I can also just set minifyEnabled false for now.